node/src/Transaction.ts

```diff
@@ -678,6 +679,22 @@ export class BaseTransaction<T extends BaseTransaction<T>> {
         return this.addAndReturn(createSMembers(key), true);
     }
 
+    /** Moves `member` from the set at `source` to the set at `destination`, removing it from the source set.
+     * Creates a new destination set if needed. The operation is atomic.
+     * See https://valkey.io/commands/smove for more details.
+     *
+     * Note: When in cluster mode, `source` and `destination` must map to the same hash slot.
```

Comment:
This is not applicable for transaction, please remove

Reply:
we decided to not add it in Transaction?

Reply:
Transaction could be routed to a single node only.
It does not protect a user from making a mistake, but this notice is more applicable to `exec` function which submits a transaction, rather than to a commands in the transaction.
If we change that decision (it is OK), we need to add the notice to all multi-key commands in transaction in all clients.